test/sagemaker_tests/pytorch/training/integration/sagemaker/test_distributed_operations.py

```diff
@@ -170,6 +170,10 @@ def test_smmodelparallel_mnist_multigpu_singlenode(ecr_image, instance_type, sag
     """
     Tests pt gpt2 command via script mode
     """
+    _, image_framework_version = get_framework_and_version_from_tag(ecr_image)
+    if Version("1.9") == Version(image_framework_version):
+        pytest.skip("Test is not supported on PyTorch 1.9")
```

Comment:
See line 45 for a better type of version check, where you can check all patch versions of PT 1.9 between "1.9" and "1.10", rather than specifically comparing to "1.9".

With this `==` method of comparison, the condition will always be ignored for versions "1.9.1" or "1.9.2", etc.

Reply:
This fixed the problem, thanks